specificatie/verblijfplaats.yaml

```diff
@@ -452,51 +357,40 @@ components:
               type: boolean
             woonplaats:
               type: boolean
-            adresregel1:
+    AdresInOnderzoek:
+      allOf:
+        - $ref: 'persoon.yaml#/components/schemas/InOnderzoek'
+        - type: object
+          properties:
+            datumVan:
               type: boolean
-            adresregel2:
+            datumIngangGeldigheid:
+              type: boolean
+            nummeraanduidingIdentificatie:
               type: boolean
-            indicatieVestigingVanuitBuitenland:
+            adresseerbaarObjectIdentificatie:
               type: boolean
-            vanuitVerblijfplaatsOnbekend:
+            functieAdres:
               type: boolean
     VerblijfplaatsOnbekendInOnderzoek:
       allOf:
         - $ref: 'persoon.yaml#/components/schemas/InOnderzoek'
         - type: object
           properties:
-            datumInschrijvingInGemeente:
-              type: boolean
-            gemeenteVanInschrijving:
+            functieAdres:
               type: boolean
-            verblijfplaatsOnbekend:
+            datumIngangGeldigheid:
               type: boolean
     LocatieInOnderzoek:
       allOf:
         - $ref: 'persoon.yaml#/components/schemas/InOnderzoek'
         - type: object
           properties:
-            adresregel1:
-              type: boolean
-            adresregel2:
-              type: boolean
-            datumInschrijvingInGemeente:
-              type: boolean
-            datumVestigingInNederland:
-              type: boolean
             datumVan:
               type: boolean
             functieAdres:
               type: boolean
-            gemeenteVanInschrijving:
-              type: boolean
-            indicatieVestigingVanuitBuitenland:
-              type: boolean
-            landVanwaarIngeschreven:
-              type: boolean
             locatiebeschrijving:
```

Comment:
bij de andere verblijfplaatstypes heb je de velden die in verblijfadres staan ook in verblijfplaats.verblijfadres.inOnderzoek toegevoegd. Alleen hier heb je locatiebeschrijving in verblijfplaats.inOnderzoek gehouden. Is dat een bewuste keuze geweest?